store/models/job_spec.go

```diff
@@ -176,11 +177,27 @@ type Initiator struct {
 // InitiatorParams is a collection of the possible parameters that different
 // Initiators may require.
 type InitiatorParams struct {
-	Schedule   Cron             `json:"schedule,omitempty"`
-	Time       Time             `json:"time,omitempty"`
-	Ran        bool             `json:"ran,omitempty"`
-	Address    common.Address   `json:"address,omitempty" storm:"index"`
-	Requesters []common.Address `json:"requesters,omitempty"`
+	Schedule   Cron           `json:"schedule,omitempty"`
+	Time       Time           `json:"time,omitempty"`
+	Ran        bool           `json:"ran,omitempty"`
+	Address    common.Address `json:"address,omitempty" storm:"index"`
+	Requesters *AddressList   `json:"requesters,omitempty"`
+}
+
+type AddressList struct {
+	Addresses []common.Address
```

Comment:
Needed for the 1-to-many it seems

Reply:
This is not a one-to-many, this is serializing as a `bytea[]` a relation would also work.